src/bp-templates/bp-nouveau/buddypress/groups/groups-loop.php

```diff
@@ -6,91 +6,165 @@
  *
  * @since   BuddyPress 3.0.0
  * @version 1.0.0
+ *
+ * @package BuddyBoss\Core
  */
 
 bp_nouveau_before_loop(); ?>
 
 <?php if ( bp_get_current_group_directory_type() ) : ?>
 	<div class="bp-feedback info">
-	<span class="bp-icon" aria-hidden="true"></span>
-	<p class="current-group-type"><?php bp_current_group_directory_type_message(); ?></p>
+		<span class="bp-icon" aria-hidden="true"></span>
+		<p class="current-group-type"><?php bp_current_group_directory_type_message(); ?></p>
 	</div>
 <?php endif; ?>
 
+<?php
+
+$cover_class        = ! bb_platform_group_element_enable( 'cover-images' ) ? 'bb-cover-disabled' : 'bb-cover-enabled';
+$meta_privacy       = ! bb_platform_group_element_enable( 'group-privacy' ) ? 'meta-privacy-hidden' : '';
+$meta_group_type    = ! bb_platform_group_element_enable( 'group-type' ) ? 'meta-group-type-hidden' : '';
+$group_members      = ! bb_platform_group_element_enable( 'members' ) ? 'group-members-hidden' : '';
+$join_button        = ! bb_platform_group_element_enable( 'join-buttons' ) ? 'group-join-button-hidden' : '';
+$group_alignment    = bb_platform_group_grid_style( 'left' );
+$group_cover_height = function_exists( 'bb_get_group_cover_image_height' ) ? bb_get_group_cover_image_height() : 'small';
+
+$group_cover_height_class = 'cover-large';
+if ( 'small' === $group_cover_height ) {
+	$group_cover_height_class = '';
+}
```

Comment:
Remove this code, add corectly here: https://github.com/buddyboss/buddyboss-platform/pull/3339/files#diff-da4134c9b4d6993bbb5f8450b6676f905ba0e0b59de3ba349c62ddd6f8a46a0eR64
`'cover-' . $group_cover_height`